CCPAConsentViewController/Classes/SourcePointClient.swift

```diff
@@ -120,6 +124,7 @@ class SourcePointClient {
             URLQueryItem(name: "accountId", value: String(accountId)),
             URLQueryItem(name: "requestUUID", value: requestUUID.uuidString),
             URLQueryItem(name: "propertyHref", value: propertyName.rawValue),
+            URLQueryItem(name: "campaignEnv", value: campaignEnv == .Stage ? "stage" : "prod"),
```

Comment:
In Android they are using stage and public variables, which one is correct

Reply:
The API expects the string `"prod"` or `"stage"` as parameters. But the user of the SDK will have to choose between `.Public` or `.Stage`. We decided to use `Public` instead of `prod` because when creating a campaign, the user will choose the environment to be `public` or `stage`.